Cards/MadGraph5_aMCatNLO/SingleT/TWminusto2L2Nu-5FS-DR1_TuneCP5_13p6TeV_amcatnlo-pythia8/TWminusto2L2Nu-5FS-DR1_TuneCP5_13p6TeV_amcatnlo-pythia8.json

```diff
@@ -0,0 +1,21 @@
+{
+    "template": "NLO_run_card.dat",
+    "template_user": [],
+    "template_vars":{
+        "parton_shower": "pythia8",
+        "ickkw": "0",
+        "maxjetflavor": "5",
+        "istr" : "1"
```

Comment:
sorry should've made this clear.

this istr is not in our template => no "$str" that can be replaced.

instead this needs to be turned on with "set run_card istr 1"